src/typings/semantic-release.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'semantic-release' {
+  interface BaseContext {
+    options: any
```

Comment:
I'm really unsure of that. I saw no places where `context.options` is used in classic semantic-release. However [here](https://github.com/semantic-release/semantic-release/blob/020ea7d7d5aa6c123fe4c97754b6b1b36c0a8e2c/index.js#L276) it seems to be set

cc @GuillaumeRahbari